backend/config.loader.php

```diff
@@ -1,15 +1,12 @@
 <?php
 
-if (
-    !empty($_SERVER['REQUEST_URI']) &&
-    substr($_SERVER['REQUEST_URI'], 0, 8) == '/branch/'
-) {
-    // for Branches
-    require_once __DIR__.'/../../../backend/config.php';
-} else if ( is_readable(__DIR__.'/config.php') ) {
-    // for Configured MASTER
+if ( is_readable(__DIR__.'/config.php') ) {
+    // for configured local
     require_once __DIR__.'/config.php';
+} else if ( is_readable(__DIR__.'/../../_backend/config.php') ) {
```

Comment:
Updated to use `master/backend/config.php`. Will need to be updated when #1282 is being worked on.